### PR TITLE
Make book read time descriptor clearer (resolves #11556)

### LINKED ISSUE #11556: Read time in book descriptions is confusing
When I find books they often contain a line in the description that says something like "This book takes 7 minutes to read."  Or "This book takes 20 minutes to read."  When I first saw this, I expected it to be literally true in terms of in-game time.  20 minutes is nearly instant.  It's only four 5 minute ticks.  Of course the reality is quite different.  The reality is that some books take multiple in-game days to read from start to finish, and yet they say it takes 20 minutes to read them and such.

So I think the simplest solution is to just remove the misleading description from the books.  However, it would be nicer if books contained a more intuitive estimate of how long they take to read from start to finish, or maybe an estimate of how long it takes to gain 1 level of skill from said book, or how long it will probably take to learn all the remaining recipes in the book, etc.

This is a minor issue for me.  I just noticed it and thought it was weird.  And I still don't know what those read times in book descriptions refer to.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/9495577-read-time-in-book-descriptions-is-confusing?utm_campaign=plugin&utm_content=tracker%2F146201&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F146201&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by NickTaylor0**
This makes it clearer that the time to read a book is not actually the time to read said book, but it is actually the time to read a chapter/section of the book. This resolves #11556

### CODE REVIEW

**Comment by KA101** [comment]
Hey, nice initiative and all, but the .pot files are updated by scripts and specifically should not be manually modified in PRs.  Mind reverting those changes?

File: src/item.cpp [hunk 1]
```diff
@@ -1001,8 +1001,8 @@ std::string item::info(bool showtext, std::vector<iteminfo> *dump, bool debug) c
                                          _("Reading this book affects your morale by <num>"),
                                          book->fun, true, (book->fun > 0 ? "+" : "")));
             }
-            dump->push_back(iteminfo("BOOK", "", ngettext("This book takes <num> minute to read.",
-                                                          "This book takes <num> minutes to read.",
+            dump->push_back(iteminfo("BOOK", "", ngettext("A chapter of this book takes <num> minute to read.",
+                                                          "A chapter of this book takes <num> minutes to read.",
```
Comment (KA101) [inline src/item.cpp]: These cpp changes are good.  Keep 'em.

File: src/player.cpp [hunk 2]
```diff
@@ -10992,8 +10992,8 @@ void player::do_read( item *book )
         if (reading->fun != 0) {
             add_msg(m_info, _("Reading this book affects your morale by %d"), reading->fun);
         }
-        add_msg(m_info, ngettext("This book takes %d minute to read.",
-                         "This book takes %d minutes to read.", reading->time),
+        add_msg(m_info, ngettext("A chapter of this book takes %d minute to read.",
+                         "A chapter of this book takes %d minutes to read.", reading->time),
```
Comment (KA101) [inline src/player.cpp]: Same here, these are keepers.

**Comment by NickTaylor-** [comment]
Oh yeah, no problem, my in-game text just didn't seem to update without editing the main one, that script must've just not run when I rebuilt it.

**Comment by NickTaylor-** [comment]
There, should be in working order now

**Comment by Zireael07** [comment]
Cool!

**Comment by KA101** [comment]
Thanks.

**Pull request merged**